src/python/pants/backend/python/typecheck/mypy/BUILD

```diff
@@ -1,22 +1,24 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_sources(dependencies=[":lockfile"])
 resource(name="lockfile", source="mypy.lock")
 
-python_tests(name="subsystem_test", sources=["subsystem_test.py"], timeout=180)
-python_tests(
-    name="rules_integration_test",
-    sources=["rules_integration_test.py"],
-    timeout=480,
-    # We want to make sure the default lockfile works for both macOS and Linux.
-    tags=["platform_specific_behavior"],
+python_sources(
+    overrides={"subsystem.py": {"dependencies": [":lockfile"]}},
 )
```

Comment:
It's overkill to use overrides for this lockfile case... all of these files transitively depend on it, so probably not worth the boilerplate to be more specific here.